spec/cypress/integration/item_spec.js

```diff
@@ -224,40 +309,54 @@ context('Item', () => {
     const summary =
       "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.";
 
-    cy.get('input#media_object_creator_0')
+    cy.get('[data-testid="resource-description-creator"]')
       .clear()
       .type(main_contributor)
       .should('have.value', main_contributor);
-    cy.get('input#display_media_object_language_0')
+    cy.get('[data-testid="resource-description-media-object[language][]"]')
+    .parent()
+      .find('input:not([readonly])') //there are two inputs, this check will select the right input
       .clear()
       .type(language)
       .should('have.value', language);
-    cy.get('textarea#abstract_0')
+    cy.get('[data-testid="resource-description-abstract"]')
       .clear()
       .type(summary)
       .should('have.value', summary);
+    //update resource description api
+    cy.intercept('POST', '/media_objects/**').as('updateResourceDescription');
     cy.get(
-      'input[name="save_and_continue"][value="Save and continue"]'
+      '[data-testid="media-object-save-btn"]'
     ).click();
+    cy.wait('@updateResourceDescription').then((interception) => {
+      expect(interception.response.statusCode).to.eq(302);
+    });
 
-    // Navigate to the preview page
-    cy.get('li.nav-item.nav-success').contains('a.nav-link', 'Preview').click();
+    
+    //Navigate to the item
+    
+    cy.intercept('GET', '**/stream').as('getmediaobject');
```

Comment:
Like the other pattern intercepting `/stream` this one should maybe be changed to `manifest.json` as well.
```suggestion
    cy.intercept('GET', '**/manifest.json').as('getmediaobject');
```